packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -98,10 +128,33 @@ function StepOne({ kycStatus }: StepOneProps) {
           <Text style={styles.description}>{t('linkBankAccountScreen.pending.description')}</Text>
         </View>
       )
-    case KycStatus.NotCreated:
-    case KycStatus.Created:
```

Comment:
So what will happen now for the notcreated and created cases? Maybe I'm missing something.

Reply:
The not created and created pieces get the 'default' from the case/switch, which is the "Verify your Identity" view